### ISSUE TITLE
Isolayer stuff should be in its own component

### ISSUE THREAD

**Issue by matthewturk** [yt-project/yt_idv#138]
Right now the isolayer logic lives inside some of the base component code, and we should probably move it to a subcomponent.  Possibly an isolayer component with a data reference to the block collection.

**Comment by chrishavlin** [comment]
agreed! i can't quite remember why it ended up there. i think there was a reason why it was more straightforward, but it'd be worth sorting that out and refactoring so that it is easy to separate out the logic.
